examples/custom-roles-analyzer/docs/contributing.md

```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```

Comment:
This file exist in the root of the repo. It's not necessary to add it here.

Reply:
Got it. Removed this file.